scripts/cleanE2e.mjs

```diff
@@ -22,15 +22,12 @@ const excludedModules = [
 ].map(dir => normalize(dir));
 
 const rootDir = resolve(dirname(fileURLToPath(import.meta.url)), '..');
+const globOptions = {absolute: true, cwd: rootDir, ignore: excludedModules};
 
 const e2eNodeModules = [
-  ...glob.sync('e2e/*/node_modules/', {cwd: rootDir}),
-  ...glob.sync('e2e/*/*/node_modules/', {cwd: rootDir}),
-]
-  .map(res => join(rootDir, res))
-  .filter(dir => !excludedModules.includes(dir))
-  .map(dir => resolve(rootDir, dir))
```

Comment:
Looks like `absolute: true` and `ignore: excludedModules` options do the same job.

Reply:
can you add `absolute: true` to https://github.com/facebook/jest/blob/8e8ee90d8dd583294fc95955e51f373d4c2fea40/scripts/build.mjs#L65-L67 as well?

Reply:
Done. Checked locally running build and tests. All works, but let’s see if CI is happy.